channels.yaml

```diff
@@ -309,6 +309,13 @@ releases:
     serverArgs: *serverArgs-v3
     agentArgs: *agentArgs-v2
     featureVersions: *featureVersions-v1
-
   #  v1.25.1+k3s1 was never released
   #  v1.25.2+k3s1 is not released through KDM yet
+  - version: v1.25.5-rc3+k3s1
+    minChannelServerVersion: v2.6.7-alpha1
```

Comment:
This should be `v2.7.2-alpha1`

Reply:
Thanks for the heads up, this is now complete 👍